test/widgets/test_widgetbox.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from libqtile.backend.x11 import xcbq
+from libqtile.bar import Bar
+from libqtile.widget import TextBox, WidgetBox
+from test.conftest import BareConfig
+
+
+def no_op(*args, **kwargs):
+    pass
+
+
+class FakeWindow:
+    class _NestedWindow:
+        wid = 10
+
+    window = _NestedWindow()
+
+
+widget_config = pytest.mark.parametrize("qtile", [BareConfig], indirect=True)
+
+
+@widget_config
+def test_widgetbox_widget(qtile):
+    qtile.conn = xcbq.Connection(qtile.display)
+    qtile.test_xeyes()
+    qtile.windows_map = {}
```

Comment:
I don't think these 3 lines are needed

Reply:
The first one is. The widgets can't create their `Drawer` objects without it. The second and third line aren't needed.

Reply:
Ah I see, OK